docs/cookbook/deployment/docker.rst

```diff
@@ -0,0 +1,33 @@
+How to deploy Sylius with Docker?
+=================================
+
+The simplest way to deploy your Sylius store with Docker is to use the template provided in the Sylius-Standard ``docker-compose.prod.yml`` configuration file.
+
+.. tip::
+
+    When using a Virtual Private Server (VPS) we recommend having at least 2GB of RAM memory.
+
+1. Install Docker on your VPS
+-----------------------------
+
+.. code-block:: bash
+
+    curl -fsSL https://get.docker.com -o get-docker.sh
+    sudo sh get-docker.sh
+
+2. Execute Docker Compose Configuration
+---------------------------------------
+
+.. code-block:: bash
+
+    export MYSQL_PASSWORD=SLyPJLaye7
```

Comment:
shouldn't it be somehow generated?

Reply:
It has to be defined by user, but in this fast route of deployment, I had to somehow define it. The root password is randomly generated, but users have to be defined. The ideal scenario states that User would use Managed DB, and the problem would not exist :)